lib/src/instance/get_instance.dart

```diff
@@ -54,12 +62,17 @@ class GetInstance {
     return find<S>(tag: tag);
   }
 
-  /// Create a new instance from builder class
-  /// Example
-  /// create(() => Repl());
+  /// Creates a new Class Instance [S] from the builder callback[S].
+  /// Every time [find]<[S]>() is used, it calls the builder method to generate
+  /// a new Instance [S].
+  ///
+  /// Example:
+  ///
+  /// ```create(() => Repl());
```

Comment:
```suggestion
  /// ```
  /// create(() => Repl());
```